src/components/Navbar/styles.scss

```
@@ -15,4 +15,21 @@
   @include respond-above(md) {
     height: rem(72px);
   }
+
+  a {
+    font-size: 16px;
+    font-weight: 500;
+    line-height: normal;
+  }
+}
+
+.link-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  &__right {
+    position: absolute;
+    right: 1rem;
```

So we don't have to big brain time.
```suggestion
    right: rem(16px);
```